hafnian/tests/test_samples.py

```diff
@@ -197,15 +203,31 @@ def test_probability_vacuum(self):
 
     def test_multimode_vacuum_state_hafnian(self):
         """Test the sampling routines by checking the samples for pure vacuum
+        using the sampler for classical states
         """
         n_samples = 100
         n_modes = 10
         sigma = np.identity(2 * n_modes)
         zeros = np.zeros(n_modes, dtype=np.int)
-        samples = hafnian_sample_state(sigma, samples=n_samples)
+        samples = hafnian_sample_classical_state(sigma, samples=n_samples)
```

Comment:
Does this make this test a lot faster?

At the same time, it probably also reduces coverage of `hafnian_sample_state`. Should we repeat this test for both functions? Could do

```python
@pytest.mark.parametrize('sample_func', [hafnian_sample_state, hafnian_sample_classical_state])
```